src/python/pants/task/fmt_task_mixin.py

```diff
@@ -7,7 +7,16 @@
 )
 
 
+class FmtGoalRegistrar(DeprecatedSkipAndDeprecatedTransitiveGoalOptionsRegistrar):
+  @classmethod
+  def register_options(cls, register):
+    super().register_options(register)
+    register("--only", type=str, default=None, fingerprint=True,
+             help="Only run the specified formatter. Currently the only accepted values are "
+                  "scalafix, or not setting any value.")
```

Comment:
Nit: we usually add the backtick ` around the value to clarify that that is what you type. Also, I don't think the comma is necessary.

Reply:
Should this be advanced for now? Generally, it should not be, but until we make this a generalized feature maybe.

Reply:
Done both